src/test/java/selenium/TestSeleniumDriver.java

```diff
@@ -1,15 +1,29 @@
-package selenium;
-
-public class TestSeleniumDriver {
-
-	public static void main(String[] args) throws InterruptedException {
-		WebDriverFacade driverFacade = new WebDriverFacade("chrome");
-		driverFacade.start("https://www.google.co.in/");
-		
-		Thread.sleep(5000);
-
-		driverFacade.stop();
-
-	}
-
-}
+package selenium;
+
+import org.openqa.selenium.By;
+
+public class TestSeleniumDriver {
+
+	public static void main(String[] args) throws InterruptedException {
+		//WebDriverFacade driverFacade = new WebDriverFacade("chrome");
+		//driverFacade.start("https://www.google.co.in/");
```

Comment:
As we are testing facebook application, why you have added chrome web page link.